### PR TITLE
Add get-rul to user command on edge

### PR DESCRIPTION
**Pull request by hkseo95**
And fix a memory leak

### CODE REVIEW

**Comment by hkseo95** [comment]
+ add to check enrolled list before sending user command

**Comment by hkseo95** [comment]
+ implement managing enrolled list

File: agent/arbiter-agent.c [hunk 1]
```diff
@@ -267,6 +267,20 @@ chamge_arbiter_agent_handle_deactivate (ChamgeDBusArbiterManager * manager,
   return TRUE;
 }
 
+static gboolean
+_is_enrolled (GList * list, const char *id)
+{
+  GList *edge;
+  for (edge = list; edge != NULL; edge = g_list_next (edge)) {
```
Comment (justinjoy) [inline agent/arbiter-agent.c]: You might want to use `g_list_find_custom ()`

Comment (hkseo95) [reply agent/arbiter-agent.c]: I will keep in mind and fix it later.

**Pull request merged**